control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -256,9 +254,10 @@ func (r *HostedControlPlaneReconciler) Reconcile(ctx context.Context, req ctrl.R
 		if err := r.Get(ctx, client.ObjectKeyFromObject(deployment), deployment); err != nil {
 			if apierrors.IsNotFound(err) {
 				newCondition = metav1.Condition{
-					Type:   string(hyperv1.KubeAPIServerAvailable),
-					Status: metav1.ConditionFalse,
-					Reason: "DeploymentNotFound",
+					Type:    string(hyperv1.KubeAPIServerAvailable),
+					Status:  metav1.ConditionFalse,
+					Reason:  "DeploymentNotFound",
```

Comment:
nit: (may be follow up) it'd be nice to have all Reason in constants as they are API and can be saw at a glance.